sky/clouds/service_catalog/oci_catalog.py

```diff
@@ -111,12 +111,16 @@ def get_default_instance_type(
     else:
         memory_gb_or_ratio = memory
 
+    def _filter_disk_type(instance_type: str) -> bool:
+        return OCI.check_disk_tier(instance_type, disk_tier)[0]
```

Comment:
```suggestion
        valid, _ = OCI.check_disk_tier(instance_type, disk_tier)
        return valid
```
code style: try not to use `[0]` as it does not indicate any information about the return type of the function. If there is a change of `OCI.check_disk_tier` function, say adding another return value, and forgetting to change here,`[0]` cannot detect the error.

Reply:
Understood, thank you! I just copied the implementation in `gcp_catalog.py`, so let me change them.